dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/LibertyCXFPropertiesTest.java

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws.fat;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.net.ssl.HttpsURLConnection;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+/*
+ * Tests the enablement of cxf.multipart.attachment
+ * and cxf.ignore.unsupported.policy property migrations
+ * from CXF 2.6.2. Test designed to work for EE9
+ *
+ * This test is using WebServiceRefFeaturesTest's infrastructure
+ * with additional codes and wsdls tailored for this test only
+ */
+@RunWith(FATRunner.class)
+@SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+public class LibertyCXFPropertiesTest {
+
+    @Server("PropertyTestServer")
+    public static LibertyServer server;
+
+    private final Class<?> c = LibertyCXFPropertiesTest.class;
+    private static final int CONN_TIMEOUT = 10;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ShrinkHelper.defaultDropinApp(server, "webServiceRefFeatures", "com.ibm.ws.test.client.stub",
+                                      "com.ibm.ws.test.wsfeatures.client",
+                                      "com.ibm.ws.test.wsfeatures.client.handler",
+                                      "com.ibm.ws.test.wsfeatures.handler",
+                                      "com.ibm.ws.test.wsfeatures.service");
+
+        server.startServer("PropertyTestServer.log");
```

Comment:
If you update the server name to `LibertyCXFPropertiesServer` you'll need to update this log file to `LibertyCXFPropertiesServer.log`